### DISCUSSION TITLE
Welcome to Hono Discussions!

### DISCUSSION THREAD

**Discussion by yusukebe** [honojs/hono#752] [Announcements]
## 👋 Welcome!

We will try to see if "Discussions" is a good way to post announcements, Q&A from developers on how to use it, etc.

Of course, it's OK to discuss questions and answers on Discord, but there was no place to write down the results. So, let's use this to store them.

**Comment by bhanuchandar-p** [comment]
How can I use Brevo service for emails

**Comment by tamiopia** [comment]
Hi Hono community 👋,

I just built Postflame, a CLI tool that automatically generates Postman collections from your Hono applications — with full support for Zod validation and @hono/zod-openapi.

NPM: https://www.npmjs.com/package/postflame

Postflame scans your Hono routes and transforms them into a structured Postman collection. When using Zod or @hono/zod-openapi, it extracts request/response schemas and examples automatically.

No more manually creating Postman docs.
